### DISCUSSION TITLE
Incorrect memory usage?

### DISCUSSION THREAD

**Discussion by ChillarAnand** [nicolargo/glances#2811] [General]
**Describe the bug**

When PyCharm is running on 8GB mac, it is consuming ~2.5 GB which is almost 30% of RAM.

<img width="1280" alt="Screenshot 2024-06-01 at 03 33 40" src="https://github.com/nicolargo/glances/assets/4463796/499e1d2d-c42c-44b8-a8a2-7f6c23e0f2b7">

However, Glances reports only 2% of RAM being used.

<img width="1280" alt="Screenshot 2024-06-01 at 03 35 32" src="https://github.com/nicolargo/glances/assets/4463796/3a6b080e-a5d8-4b7a-ab81-5e20bccebbf8">

**Comment by nicolargo** [comment]
Hi @ChillarAnand 

Process memory estimation is software dependent. PSutil (the framework behind Glances) compute process memory consuption based on the RSS aka “Resident Set Size”, this is the non-swapped physical memory a process has used. On UNIX it matches “top“‘s RES column.

Perhaps Activy Monitoring shows another metrics (like the Xmx of the Pycharm Java process) or did not take into account compression mechanism. 

Nicolas
